RELEASE_NOTES.rst

```diff
@@ -1,3 +1,90 @@
+Coala 0.10 - Polar Bear
+======================
+
+For this release we had over 170 commits contributed by 66 unique contributors
+over 2.5 months
+
+The name of this release is polar bear to reflect changes we have made to our
+release process. We have implemented a release freeze before all releases to
+to serve as a testing period for all staged changes. This should also help
+us keep releases on schedule.
+
+This release was an enhancement release which brings the usual slew of
+enhancements to documentation and api. We have started preparing the codebase
+for the big changes that are outlined in the cEPs like section inheritance
+and the next generation core.
+
+**General:**
+- various package requirements have been updated
```

Comment:
Capitalise starting words for all the points, seem to be capital in earlier releases